src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.zipcode}
+                      id="zipcode"
+                      label={`*${_('checkoutProcessForm.contact_information_zip_code')}`}
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                    <PhoneFieldItem
+                      fieldName={fieldNames.phone}
+                      label={`*${_('checkoutProcessForm.contact_information_phone')}`}
+                      placeholder={_('forms.placeholder_phone')}
+                      className="field-item--50"
+                      required
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.company}
+                      id="company"
+                      label={`${_('checkoutProcessForm.contact_information_company')}`}
+                      className="field-item--50"
+                    />
+                    <InputFieldItem
+                      type="text"
+                      label={`${_('checkoutProcessForm.contact_information_industry')}`}
+                      fieldName={fieldNames.industry}
+                      id="industry"
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+            </FieldGroup>
+            {formSubmitted ? (
+              <div className="dp-wrap-confirmation" style={containerMessageSucces}>
+                <div className="dp-msj-confirmation bounceIn" style={flextStartSyle}>
+                  <p>{_('checkoutProcessForm.success_msg')}</p>
+                </div>
+              </div>
+            ) : null}
+            <div className="dp-footer-form" style={flextStartSyle}>
+              <div className="dp-action">
+                <SubmitButton className="dp-button button-medium primary-green">
+                  {_('checkoutProcessForm.save_continue')}
+                </SubmitButton>
+              </div>
+            </div>
+          </fieldset>
+        </FormWithCaptcha>
+      ) : null}
+    </div>
+  );
+};
+
+const CheckoutProcess = ({
+  location,
+  dependencies: { planService, dopplerUserApiClient, appSessionRef },
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const { pathType, planType } = useRouteMatch().params;
+
+  const [steps, setSteps] = useState({
+    contactInformation: true,
+    billingInformation: false,
+    paymentInformation: false,
+  });
```

Comment:
This state could just be a simple string. And could be renamed to active state:
```suggestion
  const [activeStep, setActiveStep] = useState('contact-information');
```

Reply:
You can also use constants to refer to the checkout steps, something like: 
```
CHECKOUT_CONTACT_INFORMATION='contact-information'
CHECKOUT_BILLING_INFORMATION='billing-information'
```
or also something like
```
CHECKOUT_STEP_1='contact-information'
CHECKOUT_STEP_2='billing-information
```

Reply:
yes, currently I created the following:
```
const steps = {
    contactInformation: 'contact-information',
    billingInformarion: 'billing-information',
    paymentInformarion: 'payment-information',
  };
```

Are you agree?

Reply:
maybe it can be called something else, **`steps`** seems to be very general